sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DocumentCollection.java

```diff
@@ -441,6 +443,33 @@ public void setVectorEmbeddingPolicy(CosmosVectorEmbeddingPolicy value) {
         this.set(Constants.Properties.VECTOR_EMBEDDING_POLICY, value, CosmosItemSerializer.DEFAULT_SERIALIZER);
     }
 
+    /**
+     * Gets the Full Test Policy containing paths for full text search and the language specification for each path.
+     * It also contains the default language to be used.
+     *
+     * @return the FullTextPolicy
+     */
+    public CosmosFullTextPolicy getFullTextPolicy() {
+        if (this.cosmosFullTextPolicy == null) {
+            if (super.has(Constants.Properties.FULL_TEXT_POLICY)) {
+                this.cosmosFullTextPolicy = super.getObject(Constants.Properties.FULL_TEXT_POLICY,
+                    CosmosFullTextPolicy.class);
+            }
+        }
+        return this.cosmosFullTextPolicy;
+    }
+
+    /**
+     * Gets the Full Test Policy containing paths for full text search and the language specification for each path.
+     * It also contains the default language to be used.
+     *
+     * @param value the FullTextPolicy
+     */
+    public void setFullTextPolicy(CosmosFullTextPolicy value) {
```

Comment:
Its an existing pattern it-seems. 

Set -> Get -> Set -> Get
Lat Get will not see the latest set right? Is that okey?

Reply:
Its internal not public contract, based on usage may be okey

Reply:
This pattern was implemented when the SDK was ported from V2 to V4.